loading.bs

```diff
@@ -818,13 +831,23 @@ A [=request=] |browserRequest| <dfn>matches the stored request</dfn>
 1. If |browserRequest|'s [=request/url=] is not [=url/equal=] to
     |storedRequest|'s [=request/url=], return "mismatch".
 
-1. Issue: Flesh this out, along the lines of
-    https://docs.google.com/document/d/1JdJkdY7cK2rD9JXbK75sSn069c0rhF9UnLQACBZ1OtY/edit.
-
+1. [=list/For each=] header (|name|, |storedValue|) in |storedRequest|'s
+    [=request/header list=]:
+    1. If |name| doesn't name a [=content negotiation mechanism specified to be
+        usable with Variants=], return "mismatch".
```

Comment:
Also, it seems a bit up in the air as to where the UA implementor should look for specifications of Variant content negotiation mechanisms. Appendix A is one place, but presumably new or experimental HTTP headers will define them as part of their own spec? Or maybe it's OK to leave this a bit up in the air for now?

Reply:
Ah, okay, sounds good. I guess the effect of this is that SXG generators should choose the "greatest common subset" of variable headers that are supported by the UAs they're targeting. As long as UAs tend to add support for headers over time rather than remove, this should be fine.